### ISSUE TITLE
Rewrite to vue and use the most modern of modern stuff

### ISSUE THREAD

**Issue by hartman** [hartman/zonestamp#32]
I've grown to like vue a lot more than I like react, so perhaps I should rewrite the whole thing to vue. Also gives a good opportunity to redo all the date management, which could use some modernisation.

**Comment by hartman** [comment]
Date and time picker with timezone and custom styling support. Based on date-fns
https://vue3datepicker.com/

**Comment by Bam92** [comment]
@hartman: do you still have this in mind, or are you continuing with React and/or migrating to a React framework?

**Comment by hartman** [comment]
> @hartman: do you still have this in mind, or are you continuing with React and/or migrating to a React framework?

yes this is still my eventual plan, but my time is very limited, so it probably won't happen for quite some months still
